kubemarine/resources/configurations/compatibility/kubernetes_versions.yaml

```diff
@@ -137,12 +143,24 @@ compatibility_map:
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.29.0
+  v1.28.9:
+    calico: v3.28.0
+    nginx-ingress-controller: v1.10.0
+    kubernetes-dashboard: v2.7.0
+    local-path-provisioner: v0.0.26
+    crictl: v1.30.0
   v1.29.1:
     calico: v3.27.0
     nginx-ingress-controller: v1.9.6
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.29.0
+  v1.29.4:
+    calico: v3.28.0
+    nginx-ingress-controller: v1.10.0
```

Comment:
v1.10.1 is available https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.10.1